src/load/azext_load/data_plane/load_test_run/custom.py

```diff
@@ -132,6 +133,86 @@ def stop_test_run(cmd, load_test_resource, test_run_id, resource_group_name=None
     return response
 
 
+def copy_test_run_artifacts_url(cmd, load_test_resource, test_run_id, resource_group_name=None):
+    client = get_testrun_data_plane_client(cmd, load_test_resource, resource_group_name)
+    logger.info("Fetching test run copy artifacts SAS URL for test run %s", test_run_id)
+    test_run_data = client.get_test_run(test_run_id=test_run_id)
+    artifacts_container = test_run_data.get(
+        "testArtifacts", {}).get(
```

Comment:
Can we please check if this is safe for negative scenarios e.g. 
- no output artifacts, ( test run in process)
- Test run failed 

- - invalid test run id

Reply:
The SAS URL will be given out even when test run is in progress or test run failed, as long as there are input artifacts associated

Reply:
Is this the case with output artifacts as well?

Reply:
added